src/kvstore.c

```diff
@@ -61,6 +61,8 @@ struct _kvstore {
     unsigned long long key_count;          /* Total number of keys in this kvstore. */
     unsigned long long bucket_count;       /* Total number of buckets in this kvstore across dictionaries. */
     unsigned long long *dict_size_index;   /* Binary indexed tree (BIT) that describes cumulative key frequencies up until given dict-index. */
+    size_t overhead_hashtable_lut;         /* The overhead of all dictionaries. */
```

Comment:
should reset `overhead_hashtable_lut` and `overhead_hashtable_rehashing` to 0 in `kvstoreEmpty()`